src/webgpu/api/validation/createView.spec.ts

```diff
@@ -101,19 +97,48 @@ g.test('creating_texture_view_on_a_2D_non_array_texture')
     // mip level is in range
     { mipLevelCount: 1, baseMipLevel: MIP_LEVEL_COUNT - 1, _success: true },
     { mipLevelCount: 2, baseMipLevel: MIP_LEVEL_COUNT - 2, _success: true },
-    // baseMipLevel == k && mipLevelCount == 0 means to use levels k..end
-    { mipLevelCount: 0, baseMipLevel: 0, _success: true },
-    { mipLevelCount: 0, baseMipLevel: 1, _success: true },
-    { mipLevelCount: 0, baseMipLevel: MIP_LEVEL_COUNT - 1, _success: true },
-    { mipLevelCount: 0, baseMipLevel: MIP_LEVEL_COUNT, _success: false },
+    // mipLevelCount == 0 is invalid
+    { mipLevelCount: 0, baseMipLevel: 0, _success: false },
+    // baseMipLevel == k && mipLevelCount == undefined means to use levels k..end
+    {
+      mipLevelCount: undefined,
```

Comment:
mipLevleCount and arrayLayerCount below are not useful. The explicit assignment as undefined is used to be displayed on the test webpage.